eq-author-api/README.md

```diff
@@ -9,6 +9,7 @@ application.
 - [Environment Variables](#environment-variables)
 - [Run using Docker](#run-using-docker)
 - [Tests](#tests)
+- [Migrations](#migrations)
```

Comment:
I guess this will impact the docs from @jchapman68 so there may be some merging or a different format to follow